linera-explorer/src/graphql.rs

```diff
@@ -17,6 +18,24 @@ type Origin = Value;
 type UserApplicationDescription = Value;
 type ApplicationId = String;
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Notification {
+    pub chain_id: ChainId,
+    pub reason: Reason,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum Reason {
+    NewBlock {
+        height: BlockHeight,
+        hash: CryptoHash,
+    },
+    NewIncomingMessage {
+        origin: Origin,
+        height: BlockHeight,
+    },
+}
```

Comment:
Would it make sense to move this to `linera_base`, so we don't have to duplicate it here?